### DISCUSSION TITLE
why need this type of achievements

### DISCUSSION THREAD

**Discussion by DurjoyAcharya** [Schweinepriester/github-profile-achievements#656] [Q&A] [answered]
why need this type of achievements

**Comment by tyler-holm** [comment, marked as answer]
Nobody really "needs" them. They're just meant to be an incentive to get people more engaged in the GitHub community and explore new features.

If you found this response useful then please mark it as a solution.

**Comment by bm4578** [comment]
Mostly for vanity, apparently I do
